### ISSUE TITLE
Create validation set

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/champs-scalar-coupling#1]
From: https://www.kaggle.com/c/champs-scalar-coupling/data

> The training and test splits are by molecule, so that no molecule in the training data is found in the test data.

So we should remove entire molecules from the training set when creating a validation set.
